Dockerfile

```diff
@@ -16,12 +16,19 @@ RUN apt-get update -y && \
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- \
     --default-toolchain stable --profile minimal -y
 
+RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - && \
+  apt-get install -y nodejs
```

Comment:
Maybe do the npm stuff in a separate stage using a node image as base?

Reply:
That's a great idea! Thanks. I was wondering how to download `npm` more elegantly and didn't realize that a staged build is perfect for this.